src/models/GOKU.jl

```diff
@@ -40,6 +59,10 @@ function apply_latent_in(encoder::Encoder{GOKU}, pe_out)
     return (z₀_μ, θ_μ), (z₀_logσ², θ_logσ²)
 end
 
+@doc raw"""
+    apply_latent_out(decoder::Decoder{GOKU}, l̃)
+Obtains the inferred initial conditions and parameters after sampling.
```

Comment:
I would also start this in the same way as the others saying something like:

"Apply the `latent_out` layer from `encoder` to the latent variables `l̃`, returning the initial conditions (and possibly parameters, depending on the model) for the differential equation."

Maybe for the parenthesis this is better: (and depending on the model also parameters) ? I'm not sure what is better English.

Also, I'm using the word "Apply" for the different layers but I'm not sure if that is the best choice for word. Do you think it is ok or have any other suggestion?